iree/compiler/Conversion/Common/LinalgBufferizePass.cpp

```diff
@@ -816,6 +843,17 @@ static Value getAliasingBufferForResult(OpBuilder &b, SubTensorOp op,
                                      offsets, sizes, strides);
 }
 
+static SmallVector<Value> getScfForAliasingBuffers(OpBuilder &b, scf::ForOp op,
```

Comment:
Nit: If you call this method `getAliasingBufferForResult` you can just add a new entry to the existing `Case` statement (instead of a new `Case` statement) below (in `getAliasingBufferForResult(OpBuilder &b, Operation *op, BlockAndValueMapping &bvm)`)